app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -282,9 +282,11 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
 
     private fun showOtherClassifications(otherClassifications: List<Classifier.Classification>) {
         if (otherClassifications.isEmpty()) {
-            footer.visibility = View.GONE
+            otherPredictionsSeparator.visibility = View.GONE
```

Comment:
Shouldn't them be visible even when there aren't other classifications? I mean... we'll always have the main classification, right? Shouldn't we be able to benchmark it?